CONTRIBUTING.md

```diff
@@ -336,8 +336,8 @@ Here's an example:
 ```cabal
 synopsis: Add feature xyz
 packages: cabal-install
-prs: #0000
-issues: #0000 #0000
+prs: 0000
+issues: 0000 0000
```

Comment:
The `.cabal` format changelog files (according to the README, at least...) _do_ include the `#`s.

Reply:
[indeed](https://codeberg.org/fgaz/changelog-d/src/commit/0120b619ab3e4827914234674c0a3c7009642901/src/ChangelogD.hs#L315).

I guess I could make them optional

Reply:
Changed, with an explanation.